rumqttc/src/lib.rs

```diff
@@ -369,6 +369,7 @@ impl From<ClientConfig> for TlsConfiguration {
 pub struct NetworkOptions {
     tcp_send_buffer_size: Option<u32>,
     tcp_recv_buffer_size: Option<u32>,
+    tcp_nodelay: Option<bool>,
```

Comment:
```suggestion
    tcp_nodelay: bool,
```
would default the value to false anyways...